.travis.yml

```diff
@@ -113,21 +123,18 @@ before_install:
           wget -qO - https://packages.couchbase.com/ubuntu/couchbase.key | sudo apt-key add -
           sudo apt-add-repository -y 'deb http://packages.couchbase.com/ubuntu bionic bionic/main'
           sudo apt-get update && sudo apt-get install -y libcouchbase-dev
+          install: pip --disable-pip-version-check install --upgrade-strategy eager -U tox | cat
```

Comment:
Why did you move this to before_install?

Reply:
That is the order in which things execute when the job is run.